src/panel-triggers/components/AlertList/AlertList.tsx

```diff
@@ -97,15 +103,16 @@ export default class AlertList extends PureComponent<AlertListProps, AlertListSt
     return this.props.onProblemAck(problem, data);
   }
 
-  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, sortOption: string) => {
+  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, testTagFilter: boolean, sortOption: string) => {
     const textFilterLowerCase = textFilter.toLowerCase();
     let filteredProblems = this.props.problems.filter((problem: ProblemDTO) => {
       return (
         (problem.comments.toLowerCase().indexOf(textFilterLowerCase) > -1 ||
         problem.description.toLowerCase().indexOf(textFilterLowerCase) > -1) &&
         (priorityFilter === -1 || problem.severity === priorityFilter.toString()) &&
         (categoryFilter === 'all' || problem.opdata === categoryFilter) &&
-        (!maintenanceFilter || (problem.hosts.length > 0 && problem.hosts[0].maintenance_status === '0'))
+        (!maintenanceFilter || (problem.hosts.length > 0 && problem.hosts[0].maintenance_status === '0')) &&
+        (!testTagFilter || (problem.tags.length > 0 && problem.tags[0].tag !== 'test'))
```

Comment:
What if there are no  tags? The problems should be shown. So, ` || problem.tags.length == 0 ` ?
What if there are multiple tags and the second tag is `test` ? Shouldn't you use `.some()` here to find that there is no tag with key `test` ?

Reply:
In commit 74e634474759c385900228a20f037e721eb7b215.